scraper/scraper.ts

```diff
@@ -1,28 +1,480 @@
 import * as puppeteer from 'puppeteer';
 
-// import * as .. does not work for chalk, so require.
-const chalk = require('chalk');
+// Remove any html character entities from the given string
+// At this point, it only looks for 3 of them as more are not necessary
```

Comment:
Good high-level comments for functions, though try to write these in a more docstring style. For example, instead of using "//", use "/**"

```suggestion
/** 
 * Remove any html character entities from the given string
 * At this point, it only looks for 3 of them as more are not necessary
 */
```

Reply:
Added.